README.md

```diff
@@ -22,6 +22,17 @@ _The project is still heavily under construction, see the [disclaimer below](#st
 
 </div>
 
+
+## Table of Contents
+
+1. [What is BeaconKit?](#what-is-beaconkit)
+2. [Supported Execution Clients](#supported-execution-clients)
+3. [Running a Local Development Network](#running-a-local-development-network)
+4. [Multinode Local Devnet](#multinode-local-devnet)
+5. [Troubleshooting](#troubleshooting)
+6. [Status] (#Status)
```

Comment:
comes up broken

Reply:
the 'Status' right? will work on it and add a ref link to it. thanks for pointing it out.

Reply:
@ocnc i've made the necessary update and awaiting your review and possible approval